src/logic/calculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
+
+function isNumber(item) {
```

Comment:
- [ ] Kindly add a test for this module, since this is mainly the one to begin be tested